frontend/src/views/DeleteViewTemp.vue

```diff
@@ -30,8 +29,7 @@
   <script setup lang="ts">
   import { ElButton, ElDialog, ElMessage } from 'element-plus';
   import { ref } from 'vue'
-  import { deleteRequest } from '@/services/http';
-  import { removeRoleFromState } from '@/stores/roles/state';
+  import { removeRole } from '@/stores/roles/actions';
```

Comment:
You don't have to import the function directly, like this. Check in DemoView how the fetch action is being used